modules/op-using-repository-crd-with-pipelines-as-code.adoc

```diff
@@ -4,14 +4,14 @@
 
 :_content-type: REFERENCE
 [id="using-repository-crd-with-pipelines-as-code_{context}"]
-= Using the `Repository` CRD with {pac} 
+= Using the Repository custom resource definition (CRD) with {pac} 
```

Comment:
```suggestion
= Using the Repository custom resource with {pac} 
```

Reply:
The (CR) bit is defined in the abstract, no need to add it in the heading - same comment as previous also re CR vs CRD

Reply:
out of scope